<!-- This launch file does following F**king things:

1. Open a rosbag file from the specified location and Remap the topic /zoe/velodyne_points to /bakwas
2. Start triclops_stereo_pcl_bag_filters node. It takes image_raw and calibration file and generates pointcloud /zoe/bumblebee_points
3. remap topic "/zoe/bumblebee_points" to="/zoe/velodyne_points"
4. publish a static tf for bumblebee

-->

<launch>

<!-- <node pkg="rosbag" type="play" name="player" output="screen" args=" /home/anshul/logs/bumblebee/filtered_covered.bag /zoe/velodyne_points:=/bakwas"/> -->

<!--/home/anshul/logs/bumblebee/2016_Jun_16-14_06_34.bag filtered.bag 1.395467 -1.67 -0.30 1.35 -->

<!-- <node pkg="tf" type="static_transform_publisher" name="VLP16_tf_broadcaster" args="0 0 2 0 0.3488 0.034 /vehicle /velodyne 100" /> -->

<node pkg="dynamic_obstacle_tracking" type="velodyne_node" name="velodyne_node" output="screen">
<remap from="/cloud"  to="/Sensor1/pc_Sensor1"/>
<param name="frame_id" type="string" value="/odom"/>
<param name="base_frame_id" type="string" value="/base_link"/>
<param name="sensor_frame_id" type="string" value="/Sensor1"/>
</node>


<node pkg="tf" type="static_transform_publisher" name="Odom_broadcaster" args="0 0 0 0 0 0 /base_link /Body 100" />
</launch>
